build_projects/shared-build-targets-utils/Publishing/AzurePublisher.cs

```diff
@@ -15,6 +15,16 @@ namespace Microsoft.DotNet.Cli.Build
 {
     public class AzurePublisher
     {
+        public enum Products
```

Comment:
Making this enum plural seems odd since it is only used for a single value.  [https://msdn.microsoft.com/en-us/library/bb264487.aspx](https://msdn.microsoft.com/en-us/library/bb264487.aspx)